src/main/kotlin/frc/robot/Main.kt

```diff
@@ -5,7 +5,17 @@ package frc.robot
 
 import edu.wpi.first.wpilibj.RobotBase
 
+/**
+ * Do NOT add any variables to this class, or any initialization at all. Unless you know what
+ * you are doing, do not modify this file except to change the parameter class to the startRobot
+ * call.
+ */
 object Main {
+    /**
+     * Main initialization function. Do not perform any initialization here.
+     *
+     * <p>If you change your main robot class, change the parameter type.
```

Comment:
```suggestion
     * <p>If you change your main robot class, change the argument type.
```

I know wpilib uses 'parameter' here, but it's the wrong word for what they mean